src/Vault.sol

```diff
@@ -670,8 +670,7 @@ contract Vault is IERC4626, ERC20Permit, ILiquidationSource, IClaimable, Ownable
 
     _onlyVaultCollateralized(_depositedAssets, _withdrawableAssets);
 
-    uint256 _availableYield = _withdrawableAssets -
-      _convertToAssets(_depositedAssets, _depositedAssets, _withdrawableAssets, Math.Rounding.Down);
+    uint256 _availableYield = _withdrawableAssets - _depositedAssets;
```

Comment:
That's ok since we check collateralization on line 671. This function can't run if it's undercollateralized, so that's why we can simplify the math here.